Makefile

```diff
@@ -29,7 +29,7 @@ help:
 	@echo "- clean			Remove generated files"
 	@echo "- cleanall		Remove all the build artefacts"
 	@echo "- compile-catalog	Compile the translation catalog"
-	@echo "- flake8			Run flake8 checker on the Python code"
+	@echo "- flake8		Run flake8 checker on the Python code"
```

Comment:
I see, the indentation appears unregular in this github page (and in vim), but you are right it is aligned when run in the shell.